compio-driver/src/key.rs

```diff
@@ -1,40 +1,160 @@
+use std::{
+    io,
+    marker::PhantomData,
+    ops::{Deref, DerefMut},
+    pin::Pin,
+    task::Waker,
+};
+
+use compio_buf::BufResult;
+
+use crate::{OpCode, Overlapped, PushEntry, RawFd};
+
+#[repr(C)]
+pub struct RawOp<T: ?Sized> {
+    header: Overlapped,
```

Comment:
Why do we need an empty `Overlapped` on unix?

Reply:
For simplicity.

Reply:
I don't see how is that simple. It only causes confusion for me.

Reply:
There's a non-empty Overlapped on Windows.

Reply:
I know, but why change it to this when the old cfg works